scripts/west_commands/runners/bossac.py

```diff
@@ -58,6 +60,14 @@ def do_run(self, command, **kwargs):
                         'ospeed', '1200', 'cs8', '-cstopb', 'ignpar', 'eol', '255',
                         'eof', '255']
             self.check_call(cmd_stty)
+            sleep(1)
+            while(True):
```

Comment:
I think this should bail after a fixed number of retries, say 3, instead.

Reply:
That's fair enough. Changed.